src/runtime_src/core/tools/xbutil2/xbutil.cpp

```diff
@@ -66,7 +66,7 @@ R"(
     }]
   },{
     "validate": [{
-      "test": ["latency", "throughput", "cmd-chain-latency", "cmd-chain-throughput", "df-bw", "tct-one-col", "tct-all-col", "gemm", "aie-reconfig-overhead"]
+      "test": ["latency", "throughput", "npu3latency", "npu3throughput", "cmd-chain-latency", "cmd-chain-throughput", "df-bw", "tct-one-col", "tct-all-col", "gemm", "aie-reconfig-overhead"]
```

Comment:
we don't want to add "npu3latency" the user should be indifferent to npu*. The new test should work with "latency"